Gifski/VideoDropView.swift

```diff
@@ -67,6 +67,11 @@ class DropView: SSView {
 	override func draw(_ dirtyRect: CGRect) {
 		super.draw(dirtyRect)
 
+		// We only draw it when the drop view controller is the main view controller.
```

Comment:
what do you think about extracting it to a `backgroundImage` property of a `VideoDropView` and then only pass it in the controllers that do not hide the text label as well?

Reply:
I agree that would be technically better. I'm just not sure it's worth the extra complexity. VideoDropViewController is pushed in many places. Would mean a lot of duplicate code for the same result. This is only a temporary thing anyway. I'll rewrite it for SwiftUI with animations later this year.

Reply:
yeah, agree there would be some extra cost for that - I was also thinking about 2 classes, one that would be invisible and one with text & background, but that would be an excellent one for SwiftUI component composition